app/lib/features/chat/providers/notifiers/chat_room_notifier.dart

```diff
@@ -58,7 +58,7 @@ class ChatRoomNotifier extends StateNotifier<ChatRoomState> {
         state = state.copyWith(
           loading: const ChatRoomLoadingState.loading(),
         );
-        final hasMore = await timeline.paginateBackwards(20);
+        final hasMore = !await timeline.paginateBackwards(20);
```

Comment:
Not getting idea about this condition changes.

Reply:
I've upgraded the SDK to simplify the code, as a result the API has been updated to not indicate that there is more but that it reached the end. see
https://github.com/acterglobal/a3/pull/1773/files#diff-12c0480bc493ffbb94ce49b75db168b6aec7504019bcbdea587c1192ffe429c2L1128

As a result the state assumptions in the provider for `hasMore` are = `! reachedEnd`. This part of the same commit, it changed together.